src/webextension/widgets/text-area.css

```diff
@@ -17,7 +17,6 @@
 }
 
 .text-area:focus {
-  /* Note: This is reverse-engineered by looking at the mockups. */
-  border-color: #7cddee;
+  border-color: #0a84ff;
   box-shadow: 0 0 3px rgba(124, 221, 238, 0.5);
```

Comment:
Update box-shadow too with the following, please (from input.css):

```css
  /* Note: This is a best-guess from the screenshots for the Photon spec, since
     the box shadow isn't actually specified. */
  box-shadow: 0 0 3px rgba(10, 168, 255, 0.5);
```

Reply:
Good call. Thank you @jimporter